Wrappers/Python/test/test_AcquisitionGeometry.py

```diff
@@ -1609,4 +1609,21 @@ def AcquisitionGeometry_split_to_BlockGeometry(self, data, method, seed):
             
 
         for i, geo in enumerate(bg):
-            np.testing.assert_allclose(geo.angles, np.asarray(gold[i]))
\ No newline at end of file
+            np.testing.assert_allclose(geo.angles, np.asarray(gold[i]))
+    
+    def test_geometry_print_angles(self):
+        
+        AG = AcquisitionGeometry.create_Parallel2D(detector_position=[0,10])\
+            .set_panel(num_pixels=10)\
+            .set_angles(angles=range(90))\
+            .set_labels(['horizontal', 'angle'])
+        AD = AcquisitionData(np.zeros([10,90]), geometry=AG, deep_copy=False)
+        print(AD.geometry)
```

Comment:
We typically try to avoid printing in the unit tests as it clutters the output.
Maybe you could divert sys,stdout so that despite calling print the output doesn't appear?
Something like this:https://stackoverflow.com/a/34738440